client/shared/src/search/query/languageFilter.ts

```diff
@@ -0,0 +1,649 @@
+import { Literal } from './token'
+
+/**
+ * A list of all supported languages. It was extracted from
+ * https://github.com/go-enry/go-enry/blob/master/data/alias.go via
+ *
+ * curl 'https://raw.githubusercontent.com/go-enry/go-enry/master/data/alias.go' \
+ * | awk -F : '/^var LanguageByAliasMap/ { MATCH=1; next } /^}/ { exit } MATCH==1 { gsub(/^ +/, "", $2); print $2 }' \
+ * | sort | uniq
+ */
+export const allLanguages = [
```

Comment:
think we probably want to use all caps here, I'm just going by consistency with the codebase for these kinds of exported constants

```suggestion
export const ALL_LANGUAGES = [
```